shared/src/vyxal/ListHelpers.scala

```diff
@@ -467,6 +467,9 @@ object ListHelpers:
 
   def prefixes(iterable: VList): Seq[VList] = iterable.inits.toSeq.reverse.tail
```

Comment:
So uh suffixes may be fine to not work on infinite lists,

but turns out prefixes doesn't work on inf lists either.

And it should

Reply:
Not my code - ask @ysthakur